org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/MoveLogics/ScaleLogic.cs

```diff
@@ -78,8 +78,12 @@ private float GetScaleBetweenInteractors(List<IXRSelectInteractor> interactors,
                 for (int j = i + 1; j < interactors.Count; j++)
                 {
                     // Defer square root until end for performance.
-                    var distance = Vector3.SqrMagnitude(interactors[i].transform.position -
-                                                       interactors[j].transform.position);
+                    var distance = Vector3.SqrMagnitude(interactors[i].GetAttachTransform(interactable).position -
```

Comment:
I tested this a little and feels ok, but like you say scale is more sensitive with controller orientation.  And perhaps interactor positions were used originally to lesson that for scale?  But the rotation and position are already using the attach transforms, and rotation is definitely sensitive to that currently.

Reply:
@keveleigh I tested this a bit more.  My previous test was with the XRI 3 rig and their controllers in the HandsDemoScene.  It felt mostly fine with their rig and using the controllers, but I believe they are doing some smoothing of the attach transforms.

However, when testing with the MRTK3 rig (which I should've done initially), the scaling with controller far ray is quite erratic since I don't think we smooth the far ray attach transforms much if at all.  So decided to use attach transforms only as a fallback if the interactor positions appear identical.

The only case I know of where the fallback would happen is with the XRI 3 rig's hand interactor group objects.